### PR TITLE
PC-976: Init

### PR DESCRIPTION
**Pull request by ulfryk**

### CODE REVIEW

File: src/icons/char-icon/char-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as cx from 'classnames';
+import * as React from 'react';
+
+import { CV_PLAIN_ICON_PROP_TYPES, ISwCharIconProps } from './config';
```
Comment (tomecko) [inline src/icons/char-icon/char-icon.tsx]: `CV` :eye:

File: src/input/tags-input/tags-input.tsx [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+  }
+
+  constructor(props: ISwTagsInputProps<T>) {
+    super(props);
+    Bind.to(this);
+
+    if (process.env.NODE_ENV !== 'production') {
+      if ([props.suggestionSections, props.suggestions].filter(Boolean).length !== 1) {
+        // tslint:disable-next-line:no-console
+        console.error('CvTagsInput: exactly one of "suggestionSections" or "suggestions" ' +
```
Comment (tomecko) [inline src/input/tags-input/tags-input.tsx]: `Cv` :eye:

File: src/input/tags-input/tags-input.tsx [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+    );
+  }
+
+  private isEqualIgnoreCase(a: string, b: string): boolean {
+    return toLower(a) === toLower(b);
+  }
+
+}
+
+function getTypedCvTagsInput<T>() {
```
Comment (tomecko) [inline src/input/tags-input/tags-input.tsx]: search entire project for `cv`

File: src/panels/panel/panel.sass [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@import "../common"
+
+.cv-panel
```
Comment (tomecko) [inline src/panels/panel/panel.sass]: :eye:

**Comment by ulfryk** [comment]
@tomecko - all fixed in https://github.com/samwise-tech/components/pull/10/commits/9906de458ed6c266c5a9e273d117f56bb6784633

**Pull request merged**